src/snapred/backend/data/LocalDataService.py

```diff
@@ -481,28 +482,37 @@ def writeNormalizationIndexEntry(self, entry: IndexEntry):
 
     def createNormalizationIndexEntry(self, request: CreateIndexEntryRequest) -> IndexEntry:
         indexer = self.normalizationIndexer(request.runNumber, request.useLiteMode)
-        return indexer.createIndexEntry(**request.model_dump())
+        entryParams = request.model_dump()
+        entryParams["version"] = entryParams.get("version", VersionState.NEXT)
+        if entryParams["version"] is None:
+            entryParams["version"] = VersionState.NEXT
+        return indexer.createIndexEntry(**entryParams)
 
     def createNormalizationRecord(self, request: CreateNormalizationRecordRequest) -> NormalizationRecord:
         indexer = self.normalizationIndexer(request.runNumber, request.useLiteMode)
         return indexer.createRecord(**request.model_dump())
 
     def normalizationExists(self, runId: str, useLiteMode: bool) -> bool:
-        version = self.normalizationIndexer(runId, useLiteMode).currentVersion()
+        version = self.normalizationIndexer(runId, useLiteMode).getLatestApplicableVersion(runId)
```

Comment:
Again, should it be `latestApplicableVersion` or `getLatestApplicableVersion`?

Reply:
I guess `latestApplicableVersion`? it doesnt actually have a setter method so a getter doesnt make as much sense